code/game/objects/items/devices/traitordevices.dm

```diff
@@ -102,6 +102,8 @@
 	icon_state = icons_charges[charges + 1]
 
 /obj/item/teleporter/proc/attempt_teleport(mob/user, EMP_D = FALSE)
+	for(var/obj/item/grab/G in user)
+		qdel(G)
```

Comment:
this is why grabs need refactoring

Reply:
this should be gone with https://github.com/ParadiseSS13/Paradise/pull/21956